azure/packages/test/end-to-end-tests/src/test/AzureTokenFactory.ts

```diff
@@ -2,10 +2,14 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
-import { ITokenProvider } from "@fluidframework/azure-client";
+import { ITokenProvider, type ScopeType } from "@fluidframework/azure-client";
 import { InsecureTokenProvider } from "@fluidframework/test-runtime-utils";
 
-export function createAzureTokenProvider(userId: string, userName: string): ITokenProvider {
+export function createAzureTokenProvider(
+	userId: string,
+	userName: string,
+	scopes?: ScopeType[],
```

Comment:
Looks like `scopes` isn't being used, should it be passed into `new InsecureTokenProvider()` below?

Reply:
Good catch, thank you